runtime/jvmti/jvmtiThreadGroup.c

```diff
@@ -181,10 +181,30 @@ jvmtiGetThreadGroupChildren(jvmtiEnv* env,
 			threadGroupObject = *((j9object_t*)group);
 			childrenGroupsLock = J9VMJAVALANGTHREADGROUP_CHILDRENGROUPSLOCK(currentThread, threadGroupObject);
 			childrenGroupsLock = (j9object_t)vmFuncs->objectMonitorEnter(currentThread, childrenGroupsLock);
```

Comment:
is the monitor check for `childrenGroupsLock` duplicated?

Reply:
Hmm, it probably was added during rebasing, removed.